core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -165,6 +174,7 @@ object ConsumerPerformance {
 
     totalMessagesRead.set(messagesRead)
     totalBytesRead.set(bytesRead)
+    joinTime.set(joinEnd - joinStart)
```

Comment:
Currently, this perf tool does not support multiple consumer instances in one consumer group for new consumer, so is it possible that there will be more than one rebalance?